Fastlane/deployment_lanes.rb

```diff
@@ -162,7 +162,7 @@
     sh "git branch #{branch_name} origin/main"
     sh "git checkout #{branch_name}"
 
-    sh "git merge -X theirs #{latest_release_tag}" unless is_hotfix
```

Comment:
Should we, though? Isn't there a way to fix it?

Reply:
I wish there is a way, but I have no clue where this is caused